src/test/tests.ts

```diff
@@ -896,7 +893,15 @@ describe('Server', function () {
       //do nothing
     });
 
-    client.unsubscribe(subId);
+    setTimeout(() => {
+      // TODO
+      // Hagaico I had the need to add this timeout here
```

Comment:
is `onOperation` callback being called?
if it does, then i think we should fix it..
if its not called, then we need to swap the test to make sure it's not being called..